SPECS/kernel/config

```diff
@@ -1073,6 +1073,7 @@ CONFIG_DEVICE_MIGRATION=y
 CONFIG_ARCH_ENABLE_HUGEPAGE_MIGRATION=y
 CONFIG_ARCH_ENABLE_THP_MIGRATION=y
 CONFIG_CONTIG_ALLOC=y
+CONFIG_PCP_BATCH_SCALE_MAX=5
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/dde5e5343d2be76aba2f8d12afb2e7120b42c94e